src/vcpkg/install.cpp

```diff
@@ -1003,6 +1019,50 @@ namespace vcpkg::Install
             {
                 extended_overlay_ports.push_back(paths.builtin_ports_directory().native());
             }
+
+            std::string hash;
+            {
```

Comment:
I'm not sure we want to calculate the hash if the `check-stamp` option isn't passed